### ISSUE TITLE
Installing

### ISSUE THREAD

**Issue by callumacrae** [lambdatoast/elm.vim#14]
How do I actually install this? I'm not great at vim, and the usual stuff I do with pathogen doesn't seem to load the plugin.

Sorry for being a noob :)

**Comment by ifraixedes** [comment]
I got installed it with [Vundle](https://github.com/gmarik/Vundle.vim)

**Comment by kasbah** [comment]
Mine is installed with Pathogen. Just git cloned `elm.vim` into `~/.vim/bundle`.

**Comment by lambdatoast** [comment]
Yes, easiest way to manage plugins is to install vim pathogen. And after you have pathogen installed, installing a plugin is a matter of just cloning its repo into your `~/.vim/bundle`

**Comment by callumacrae** [comment]
Cloned into `~/.vim/bundle`, didn't work :/

**Comment by lambdatoast** [comment]
@callumacrae Did you install pathogen? You must install pathogen, and then you should have e.g. `~/vim/bundle/elm.vim`.

**Comment by callumacrae** [comment]
Yes, and I've installed other plugins through it, and I ran `:Helptags`.

**Comment by lambdatoast** [comment]
It doesn't load the syntax highlighting or anything?

**Comment by callumacrae** [comment]
No syntax highlighting, no docs, no commands. It's as if it just isn't loaded.

**Comment by ludvigsen** [comment]
I'm having the same problem as @callumacrae, I installed it via Vundle.

**Comment by thelastinuit** [comment]
Same here. No highlight.

**Comment by ludvigsen** [comment]
Just if there are other people having issues, I got it working by installing pathogen, so it seems like it somehow works with pathogen but not vundle.

**Comment by fpguy** [comment]
Not working with pathogen either

**Comment by fpguy** [comment]
worked after few tries...

**Comment by pneisen** [comment]
I am installing via pathogen. I believe the syntax highlighting is working, but I can't get the commands to work.

**Comment by kitofr** [comment]
I managed to install the vim package with `vundle` using

```.vimrc
Bundle 'lambdatoast/elm.vim'
```

**Comment by damionvega** [comment]
Everyone should really look into vim-plug: https://github.com/junegunn/vim-plug

**Comment by gabrielflorit** [comment]
Also there's another elm vim plugin worth looking into: https://github.com/ElmCast/elm-vim

**Comment by viswaug** [comment]
I tried this plugin with Vundle by adding (Plugin 'elmcast/elm-vim.git') to my .vimrc and running ":PluginInstall", but couldn't get it to work.

The error on running ":ElmRepl" is "E492: Not an editor command: ElmRepl"

The ":PluginList" command lists the plugin in. The files have been checkout into the ~/.vim/bundle folder. But the ":scriptnames" command does not list the "elm.vim" file in the plugin folder as being loaded. The ":func ElmRepl" errors with "E123: Undefined function: ElmRepl".

Any way to troubleshoot why the plugin functions are not getting loaded? Any log files anywhere i should look into?

**Comment by gabrielflorit** [comment]
@viswaug You should file this issue in the [ElmCast/elm-vim](https://github.com/ElmCast/elm-vim) repository.

**Comment by viswaug** [comment]
@gabrielflorit oops. I actually meant to say that used (Plugin 'lambdatoast/elm.vim.git') in my .vimrc file. And this issue is relevant here. I did try the other plugin too as someone recommended it here. Any ideas why the functions from the plugin are not loading? Any way to debug it?

**Comment by viswaug** [comment]
[vimlog.txt](https://github.com/lambdatoast/elm.vim/files/137091/vimlog.txt)
Ok, I am back with more info. I am attaching my vim log file above. It doesn't seem to be loading the "elm.vim" file in the "~/.vim/bundle/elm.vim/ftplugin" folder. It does seem to be looking for "~/.vim/bundle/elm.vim/ftplugin.vim" from the logs. So, I tried creating a symlink to the "~/.vim/bundle/elm.vim/ftplugin/elm.vim" file at "~/.vim/bundle/elm.vim/ftplugin.vim" and that doesn't seem to load it either. But, if i source it with ":so ~/.vim/bundle/elm.vim/ftplugin/elm.vim", the functions are loaded and it works.

**Comment by patricksimpson** [comment]
In the elm.vim directory run: `ln -s ftplugin plugin` The plugin directory is needed for vundle/bundle installs. I hope this helps. This needs fixed in the repository.

**Comment by viswaug** [comment]
@patricksimpson that fixed it for me. Thanks a bunch. Appreciate it

**Comment by patricksimpson** [comment]
@viswaug No problem, I opened a PR that adds this symlink. Though updating vundle may solve this issue as well.
